app/models/restaurant.rb

```diff
@@ -1,4 +1,6 @@
 class Restaurant < ApplicationRecord
   validates_presence_of :name, presence: true
   validates_presence_of :address, presence: true
+
+  belongs_to :rest_category
```

Comment:
Style comment: associations are placed before validations, so the "belongs_to" line should really be at the top.